src/systems/odometry_publisher/OdometryPublisher.cc

```diff
@@ -95,7 +96,7 @@ class gz::sim::systems::OdometryPublisherPrivate
   public: math::Pose3d lastUpdatePose{0, 0, 0, 0, 0, 0};
 
   /// \brief Current timestamp.
-  public: math::clock::time_point lastUpdateTime;
+  public: std::chrono::steady_clock::time_point lastUpdateTime;
```

Comment:
@chapulina, I chose to change this instead of including
 `gz/math/DiffDriveOdometry.hh` (or `gz/math/GaussMarkovProcess.hh`, `gz/math/Stopwatch.hh`), since I'm not not sure we want to encourage/continue using the type alias defined there.

Reply:
Good call 👍🏽